packages/block-library/src/template-part/index.php

```diff
@@ -106,16 +106,20 @@ function render_block_core_template_part( $attributes ) {
 	// is set in `wp_debug_mode()`.
 	$is_debug = WP_DEBUG && WP_DEBUG_DISPLAY;
 
-	if ( is_null( $content ) && $is_debug ) {
-		if ( ! isset( $attributes['slug'] ) ) {
-			// If there is no slug this is a placeholder and we dont want to return any message.
-			return;
+	if ( is_null( $content ) ) {
+		if ( $is_debug ) {
+			if ( ! isset( $attributes['slug'] ) ) {
```

Comment:
By flipping the state and combing this `if` with the one above, the `sprintf` can move up so that there's only 1 `return '';` in this branch. It's a tiny simplification that reduces the number of nested branches (for readability).

```
	if ( is_null( $content ) ) {
		if ( $is_debug && isset( $attributes['slug'] ) ) {
			return sprintf(
				/* translators: %s: Template part slug. */
				__( 'Template part has been deleted or is unavailable: %s' ),
				$attributes['slug']
			);
		}

		return '';
	}
```

Reply:
Incorporated, good catch!